public/src/modules/taskbar.js

```diff
@@ -156,7 +156,7 @@ define('taskbar', ['benchpress', 'translator'], function (Benchpress, translator
 				.addClass(data.options.className)
 				.html('<a href="#"' + (data.options.image ? ' style="background-image: url(\'' + data.options.image + '\'); background-size: cover;"' : '') + '>' +
 					(data.options.icon ? '<i class="fa ' + data.options.icon + '"></i> ' : '') +
-					'<span component="taskbar/title">' + title + '</span>' +
+					'<span aria-label=' + title + ' component="taskbar/title">' + title + '</span>' +
```

Comment:
Needs to have `"` on either side

Reply:
thanks for catching that @pitaj, 
@preetykumar thanks for submitting this improvement :+1: